fedimint-client-legacy/src/lib.rs

```diff
@@ -265,34 +265,6 @@ impl<T: AsRef<ClientConfig> + Clone + Send> Client<T> {
         self.config.clone()
     }
 
-    /// Verifies the config using the federation id
-    pub async fn verify_config(&self, id: &FederationId) -> Result<()> {
```

Comment:
We remove this and the tests because validation now happens when the config is downloaded initially.